packages/kbn-es/src/ess_resources/README.md

```diff
@@ -0,0 +1,49 @@
+# Elasticsearch Serverless Resources
+The resources in this directory are used for seeding Elasticsearch Serverless (ESS) images with users, roles and tokens for SSL and authentication. ESS requires file realm authentication, so we will bind mount them into the containers at `/usr/share/elasticsearch/config/`.
+
+## Users
+
+### Default user
+
+The default superuser authentication to login to Kibana is:
+
+```
+username: elastic_serverless
+password: changeme
+```
+
+### Adding users
```

Comment:
Question: Once we create a new user, what will be the best way to run tests with that user?

Reply:
Not sure what the use case would be to test things with other user than the system_indices user or the elastic_serverless one but if thats the case, after creating the user, we can override the serverless config (providing a different user to run those specific tests with) or by creating a service with authOverride like for example [`test/functional/services/remote_es/remote_es.ts`](https://github.com/elastic/kibana/blob/main/test/functional/services/remote_es/remote_es.ts)